extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+
+:host {
+    align-self: start;
+    --correct-color: #5BBA74;
```

Comment:
Are these allowed to be overridden?  I didn't think they were supposed to be

Reply:
Ah yeah I need to add !important to most of this CSS